flume/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: flume
+
+## Overview
+
+This check monitors [flume][1].
+
+## Setup
+
+### Installation
+
+To install the flume check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
```

Comment:
Move link URLs to bottom of file, and reference by number (like the link in line 5).